examples/integration/src/codegen/server.rs

```diff
@@ -39,6 +47,29 @@ impl BookServiceCodeGen {
                 res.encode_to_vec()
             })
         });
+
+        let serv = Arc::clone(&service);
```

Comment:
Let's improve legibility of code and variable names for an easier future onboarding!
```suggestion
        let service = Arc::clone(&service);
```